app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    path = parsed_command[-1].split("/")
```

Comment:
1. instead of using indexes, better use unpacking.
2. `os.path.split()` will be better here since it will correctly split on all systems